galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.cloud.manager/src/main/java/dev/galasa/cloud/CloudContainerEnvProp.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.cloud;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * Cloud Container - Environment Properties
+ * 
+ * If any environment properties are required to be passed to the container at startup, they can be specified via here.
+ * 
+ */
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
should this be annotated CloudManagerField - I won't raise a comment for each of them